packages/core/package.json

```diff
@@ -10,34 +10,24 @@
   "types": "dist/esm/src/index.d.ts",
   "files": [
     "dist",
-    "internal.js",
-    "internal.d.ts",
-    "testing.js",
-    "testing.d.ts",
     "src"
   ],
   "dependencies": {
-    "@ethersproject/abi": "5.6.1",
-    "@ethersproject/abstract-provider": "5.6.0",
-    "@ethersproject/bignumber": "5.6.0",
-    "@ethersproject/constants": "5.6.0",
-    "@ethersproject/contracts": "5.6.0",
-    "@ethersproject/providers": "5.6.2",
-    "@ethersproject/wallet": "5.6.0",
     "@metamask/detect-provider": "^1.2.0",
     "@uniswap/token-lists": "^1.0.0-beta.27",
-    "ethers": "5.6.2",
     "fetch-mock": "^9.11.0",
     "lodash.merge": "^4.6.2",
     "lodash.pickby": "^4.6.0",
     "nanoid": "3.1.22",
     "react-flat-providers": "^2.1.0"
   },
   "peerDependencies": {
-    "react": "*"
+    "react": "*",
+    "ethers": "*"
   },
   "devDependencies": {
     "@ethereum-waffle/provider": "4.0.0-alpha.0",
+    "@ethersproject/abstract-provider": "^5.6.1",
```

Comment:
Is it not included in ethers?

Reply:
Not everything is exported in a nice way. Anyway we need only types from `@ethersproject/abstract-provider`, so I think this is ok.